app/(authorized)/lines/(drawer)/menu.tsx

```
@@ -0,0 +1,41 @@
+import { Text, useThemeColor } from "@/components/Themed";
+import { API } from "@/services/api";
+import { routes } from "@/utils/utils";
+import { MaterialIcons } from "@expo/vector-icons";
+import { useMutation } from "@tanstack/react-query";
+import { useRouter } from "expo-router";
+import { Box } from "native-base";
+import { FC } from "react";
+import { ActivityIndicator, Pressable } from "react-native";
+
+type MenuPropsType = {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)